### PR TITLE
Keep consistent defines naming scheme

### PR DESCRIPTION
**Pull request by Maxez**
## Description

Changed names of some defines.

## Motivation and Context

Paweł asked to change names of those defines.
[JIRA: PD-142](https://jira.phoenix-rtos.com/browse/PD-142)

## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [ ] New feature (non-breaking change which adds functionality)
- [ ] Breaking change (fix or feature that would cause existing functionality to change)

## How Has This Been Tested?

- [ ] Already covered by automatic testing.
- [ ] New test added: (add PR link here).
- [ ] Tested by hand on: (list targets here).

## Checklist:

- [ ] My change requires a change to the documentation.
- [ ] I have updated the documentation accordingly.
- [ ] I have added tests to cover my changes.
- [x] All new and existing linter checks and tests passed.
- [x] My changes generate no new compilation warnings for any of the targets.

## Special treatment

- [ ] This PR needs additional PRs to work (list the PRs, preferably in merge-order).
- [x] I will merge this PR by myself when appropriate.

### CODE REVIEW

**Review by agkaminski — APPROVED** [review]

**Review by HBuczynski — CHANGES_REQUESTED** [review]
If you want to be consistent with new name scheme, update also the rest of defines.
After your changes, there is a mix of conventions in `hal/*/*/config.h`
For example:
https://github.com/phoenix-rtos/plo/blob/75e3f82897df2649d8b7d1b2e69120d6275ec02f/hal/armv7m/imxrt/117x/config.h#L29-L35

**Comment by Maxez** [comment]
> If you want to be consistent with new name scheme, update also the rest of defines.
> After your changes, there is a mix of conventions in `hal/*/*/config.h`
> For example:
> 
> https://github.com/phoenix-rtos/plo/blob/75e3f82897df2649d8b7d1b2e69120d6275ec02f/hal/armv7m/imxrt/117x/config.h#L29-L35

Done.

File: cmds/bitstream.c [hunk 1]
```diff
@@ -31,7 +31,7 @@ static int cmd_bitstream(int argc, char *argv[])
 	ssize_t res;
 	handler_t handler;
 	u8 buff[SIZE_MSG_BUFF];
-	addr_t offs = 0, addr = BITSREAM_ADDR;
+	addr_t offs = 0, addr = ADDR_BITSREAM;
```
Comment (agkaminski) [inline cmds/bitstream.c:34]: `BITSTREAM` perhaps?

Comment (Maxez) [reply cmds/bitstream.c:34]: Fixed. I found another typo: `MICORN_MT25QL01GBBB` changed to `MICRON_MT25QL01GBBB`

Comment (agkaminski) [reply cmds/bitstream.c:34]: All appearances had this typo

**Review by agkaminski — APPROVED** [review]

**Review by HBuczynski — DISMISSED** [review]

**Review by HBuczynski — APPROVED** [review]

**Pull request merged**